src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java

```diff
@@ -13,41 +22,28 @@ private static int getExpAtLevel(final int level) {
         return (9 * level) - 158;
     }
 
+    /**
+     * Calculates the total amount of experience a Player has without the level system.
+     * @param player The Player whose experience should be calculated
+     * @return The total experience of the provided Player
+     */
     public static int getTotalExperience(final Player player) {
-        int exp = Math.round(getExpAtLevel(player.getLevel()) * player.getExp());
-        int currentLevel = player.getLevel();
-
-        while (currentLevel > 0) {
-            currentLevel--;
-            exp += getExpAtLevel(currentLevel);
-        }
-        if (exp < 0) {
-            exp = Integer.MAX_VALUE;
-        }
-        return exp;
+        return player.getTotalExperience();
```

Comment:
I cant remember the exact reason but I did this on purpose. I belive that player.getTotalExperiance was sometimes wrong? I cant quite recall exactly

Reply:
I guess that could be the case (https://bukkit.org/threads/p-gettotalexperience-acting-weird.407546/#post-3331133), but I think it works correctly on newer versions of Spigot. It depends if you want to keep 1.8 - 1.12 support

Reply:
Ah ye, that sounds about right. Keep it has it was before. Im going to try and keep 1.8-1.12 depending on how much of a headache it gives me